code/script.js

```diff
@@ -1,9 +1,19 @@
-// A function that adds and remove the class "active" on the section you click on.
-// We haven't really talked about ´this´ yet, but we will... ;)
-// console.log(this) to see how it works
-function toggle() {
-  this.classList.toggle("active")
+const accordion = document.getElementsByClassName ('contentBx');
```

Comment:
Nice that you used getElementsByClassName, I used getElementsById. You're teaching me a lot of things here :)